build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/precommit/ThirdPartyAuditTask.java

```diff
@@ -218,7 +218,8 @@ public void runThirdPartyAudit() throws IOException {
             if (bogousExcludesCount != 0 && bogousExcludesCount == missingClassExcludes.size() + violationsExcludes.size()) {
                 logForbiddenAPIsOutput(forbiddenApisOutput);
                 throw new IllegalStateException(
-                    "All excluded classes seem to have no issues. This is sometimes an indication that the check silently failed"
+                    "All excluded classes seem to have no issues. This is sometimes an indication that the check silently failed "
+                        + "or that exclusions are configured unnecessarily"
```

Comment:
I added this since I spent way too long trying to figure out why this would silently fail ... In my case it didn't silently fail, but instead had configuration from before the upgrade which is is no longer needed.